### ISSUE TITLE
sphinx+rst2pdf fails with automodule directive

### ISSUE THREAD

**Issue by dkriegner** [rst2pdf/rst2pdf#566]
I am trying to build API documention of one of my projects using sphinx+rst2pdf. Since updating to sphinx-1.4.1 I obtained errors upon building the pdf documention. 

I tried to narrow the problem down to a minimal example with the following documentation code.

```
test documentation
===================

test

API-documentation
=================

.. automodule:: rst2pdf
```

and sphinx _conf.py_ file

```
# -*- coding: utf-8 -*-

# Add any Sphinx extension module names here, as strings.
extensions = [
    'sphinx.ext.autodoc',
    'rst2pdf.pdfbuilder']

# The master toctree document.
master_doc = 'index'

# General information about the project.
project = u'test'
copyright = u'2016, authors'

# The short X.Y version.
version = '1'
# The full version, including alpha/beta/rc tags.
release = version

# -- Options for PDF output --------------------------------------------------
pdf_documents = [
    ('index',
     u'test',
     u'test',
     u'DK'),
]
```

I also attach the files:
[doc.zip](https://github.com/rst2pdf/rst2pdf/files/228079/doc.zip)

When I try to build this with sphinx-1.4.1 and rst2pdf master (commit af7cabb) I obtain the following:

```
sphinx-build -b pdf doc/source/ build/sphinx/
Running Sphinx v1.4.1
making output directory...
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [pdf]: targets for 1 source files that are out of date
updating environment: 1 added, 0 changed, 0 removed
reading sources... [100%] index                                                                                                                      
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
processing test... index [ERROR] pdfbuilder.py:130 too many values to unpack
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/rst2pdf-0.93.dev_r0-py2.7.egg/rst2pdf/pdfbuilder.py", line 122, in write
    appendices=opts.get('pdf_appendices', self.config.pdf_appendices) or [])
  File "/usr/lib64/python2.7/site-packages/rst2pdf-0.93.dev_r0-py2.7.egg/rst2pdf/pdfbuilder.py", line 209, in assemble_doctree
    index_nodes=genindex_nodes(genindex)
  File "/usr/lib64/python2.7/site-packages/rst2pdf-0.93.dev_r0-py2.7.egg/rst2pdf/pdfbuilder.py", line 385, in genindex_nodes
    for entryname, (links, subitems) in entries:
ValueError: too many values to unpack
FAILED
build succeeded.
```

while with sphinx-1.2.2 it generates the pdf output as expected:

```
sphinx-build -b pdf doc/source/ build/sphinx/
Making output directory...
Running Sphinx v1.2.2
loading pickled environment... not yet created
building [pdf]: targets for 1 source files that are out of date
updating environment: 1 added, 0 changed, 0 removed
reading sources... [100%] index                                                                                                            
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
processing test... index  py-modindex

resolving references...
done
writing test... done
build succeeded.
```

html output is generated successfully with both versions. When removing the _automodule_ line the pdf out is also generated using sphinx-1.4.1

**Comment by KonstantinShemyak** [comment]
Fixed by pull request #576.

**Comment by akrabat** [comment]
Resolved with the merge of #619.
